commcare_connect/form_receiver/processor.py

```diff
@@ -271,6 +272,7 @@ def process_deliver_unit(user, xform: XForm, app: CommCareApp, opportunity: Oppo
         and not user_visit.flagged
     ):
         user_visit.update_status(VisitValidationStatus.approved)
+        user_visit.review_status = VisitReviewStatus.agree
```

Comment:
are we able to distinguish visits that were auto marked as agreement, and those manually approved by an enterprise user?

Reply:
We can distinguish between auto marked as agreement and manually approved by using the review_created_on field. That field is not present on auto-marked as agreement visits. That field is also used to indicate that manual approval is required.